java/runtime/src/main/java/org/ray/runtime/RayNativeRuntime.java

```diff
@@ -147,6 +147,16 @@ public void setResource(String resourceName, double capacity, UniqueId nodeId) {
     nativeSetResource(nativeCoreWorkerPointer, resourceName, capacity, nodeId.getBytes());
   }
 
+  @Override
+  public Object getAsyncContext() {
+    throw new UnsupportedOperationException();
```

Comment:
We can't just throw exceptions here. Because, in drivers, the runtime is `RayNativeRuntime`.

Reply:
Done.